src/initStackArgs.ts

```diff
@@ -7,11 +7,16 @@ export async function initStackArgs(argv: Arguments): Promise<number> {
     Type: "AWS::CloudFormation::WaitConditionHandle"
     Properties: {}`;
 
-  const stackArgs = `# REQUIRED SETTINGS:
+  const stackArgs = `# INITIALIZED STACK ARGS
+# $imports:
+#   environment: env:ENVIRONMENT
+
+# REQUIRED SETTINGS:
 StackName: <string>
 Template: ./cfn-template.yaml
 # optionally you can use the yaml pre-processor by prepending 'render:' to the filename
 # Template: render:<local file path or s3 path>
+# ApprovedTemplateLocation: s3://your-bucket/
```

Comment:
If we have a pending template location, it would be easier to lock the whole bucket so that no files could be edited/overwritten.